crates/libtiny_tui/src/msg_area/mod.rs

```diff
@@ -74,7 +74,9 @@ impl MsgArea {
         self.lines_height = None;
 
         self.update_total_visible_lines();
-        self.recalculate_scroll(old_height, old_total_lines.unwrap());
+        if let Some(old_total_lines) = old_total_lines {
+            self.recalculate_scroll(old_height, old_total_lines);
+        }
```

Comment:
Let's also revert this. As mentioned in my other comment, this code is currently buggy and removing the `unwrap` will hide the bug. We should fix this in another PR.